src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties

```diff
@@ -1,4 +1,6 @@
 at.clause.order=Block tags have to appear in the order ''{0}''.
+inlineReturn.javaDoc=Forbidden inline @return.
+inlineReturn.javaDoc.wrongMemberType=@return is placed on the wrong type of member.
```

Comment:
@octylFractal @pbludov To me, this is not clear from the original issue. I was under the impression we were suppressing a false violation. But are we expanding Check to catch more things? It seems issue is not clear in definition of what to do and it should closely match the PR. The issue and title is what is used to put what we are changing in the release notes.

To me, check is only checking the summary sentence and that another check should be made to identify tags (inline or not) that are used in the wrong place. We need to keep checks from exploding with too much functionality. We have already seen this in other checks that it becomes a burden and they need to be rewritten.

Reply:
That was my feeling as well, but I figured that @pbludov knew better that I did.

Reply:
I agree with @rnveach. The commit message should be changed to something like "SummaryJavadoc shouldn't report javadocs with inline return tag"

Reply:
Please remove new messages, validation should stay be done by same messages.